velox/common/caching/tests/AsyncDataCacheTest.cpp

```diff
@@ -505,6 +505,9 @@ void AsyncDataCacheTest::loadNFiles(
   for (auto i = 0; i < numLoads; i++) {
     semaphore.acquire();
   }
+  while (cache_->ssdCache()->writeInProgress()) {
+    std::this_thread::sleep_for(std::chrono::milliseconds(100)); // NOLINT
+  }
 }
 
```

Comment:
#ifndef NDEBUG
DEBUG_ONLY_TEST_F(AsyncDataCacheTest, ttl)

You can remove "#ifndef NDEBUG", DEBUG_ONLY_TEST_F only works on debug build.

Reply:
Hi @xiaoxmeng  Revise. Can you take a look again?